eventuals/task.h

```diff
@@ -261,29 +314,40 @@ struct _TaskFromToWith final {
     void Dispatch(
         Action action,
         std::optional<MonostateIfVoidOr<From_>>&& from = std::nullopt,
-        std::optional<std::exception_ptr>&& exception = std::nullopt) {
+        std::optional<
+            std::conditional_t<
+                std::tuple_size_v<Catches_>,
+                apply_tuple_types_t<std::variant, Catches_>,
+                std::monostate>>&& error = std::nullopt) {
       CHECK_EQ(value_or_dispatch_.index(), 1u);
 
       std::apply(
-          [&](auto&... args) {
+          [&](auto&&... args) {
             std::get<1>(value_or_dispatch_)(
                 action,
-                std::move(exception),
-                args...,
+                std::move(error),
+                std::forward<decltype(args)>(args)...,
```

Comment:
Please update `generator.h` and `transformer.h` with this change too! Consistency!